diff --git a/install/zenohcxx.pc.in b/install/zenohcxx.pc.in
index 4493299..d6a08e9 100644
--- a/install/zenohcxx.pc.in
+++ b/install/zenohcxx.pc.in
@@ -4,4 +4,4 @@ Name: @PROJECT_NAME@
 Description: @CMAKE_PROJECT_DESCRIPTION@
 URL: @CMAKE_PROJECT_HOMEPAGE_URL@
 Version: @PROJECT_VERSION@
-Cflags: -I${prefix}/@CMAKE_INSTALL_INCLUDEDIR@
\ No newline at end of file
+Cflags: -I/@CMAKE_INSTALL_FULL_INCLUDEDIR@
